calendar-bundle/src/Resources/contao/classes/Events.php

```diff
@@ -116,7 +117,7 @@ protected function getAllEvents($arrCalendars, $intStart, $intEnd)
 		foreach ($arrCalendars as $id)
 		{
 			// Get the events of the current period
-			$objEvents = CalendarEventsModel::findCurrentByPid($id, $intStart, $intEnd);
+			$objEvents = CalendarEventsModel::findCurrentByPid($id, $intStart, $intEnd, array(), $blnFeatured);
```

Comment:
This would be `findCurrentByPid($id, $intStart, $intEnd, array('showFeatured' => $blnFeatured));` then.